src/vs/workbench/contrib/mergeEditor/browser/mergeEditorInputModel.ts

```diff
@@ -350,15 +351,15 @@ class WorkspaceMergeEditorInputModel extends EditorModel implements IMergeEditor
 		return this.resultTextFileModel.resource;
 	}
 
-	async save(): Promise<void> {
-		await this.resultTextFileModel.save();
+	async save(options?: ITextFileSaveOptions): Promise<void> {
+		await this.resultTextFileModel.save(options);
```

Comment:
Its important to pass on the save options to the file model as it will reason about it. For example, save formatters may only run on explicit save, not auto save.